# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/glibc22/glibc-2.2.2-mtrace-intl-perl.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


Don't run this check if no perl is found on the system

--- ./intl/Makefile~	Fri Feb  9 19:04:07 2001
+++ ./intl/Makefile	Mon Apr 30 14:21:10 2001
@@ -53,6 +53,7 @@
 
 ifeq (no,$(cross-compiling))
 ifeq (yes,$(build-shared))
+ifneq (no,$(PERL)) 
 ifneq ($(strip $(MSGFMT)),:)
 tests: $(objpfx)mtrace-tst-gettext $(objpfx)tst-translit.out \
        $(objpfx)tst-gettext2.out
@@ -65,6 +66,7 @@
 	$(SHELL) -e $< $(common-objpfx) $(common-objpfx)intl/
 $(objpfx)tst-gettext2.out: tst-gettext2.sh $(objpfx)tst-gettext2
 	$(SHELL) -e $< $(common-objpfx) $(common-objpfx)intl/
+endif
 endif
 endif
 endif
